src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -186,11 +194,15 @@
                                                :from 0
                                                :size 10}))))}
         :typeaheadDisplay (fn [result]
-                            ;; we intentionally do not encode the result value here, because it is already
-                            ;; encoded from the server.
-                            (.text (js/$ (str "<div>" (aget result "value") "</div>"))))
+                            (let [sugg-object (js->clj (aget result "value"))
+                                  suggestion (get sugg-object "suggestion")]
+                              (.text (js/$ (str "<div>" suggestion "</div>")))))
```

Comment:
The call to `.text()` is weird because it strips out the HTML, which means we add it with the `div` elements and then immediately remove it. I guess I would prefer either:
```cljs
(-> (str "<div>" (encode suggestion) "</div>") js/$)
```
or
```cljs
(-> suggestion js/$ (js-invoke "text"))
```
(no need to use my syntax, just giving an example)